Nos/Views/Onboarding/BuildYourNetworkView.swift

```diff
@@ -9,29 +9,42 @@ struct BuildYourNetworkView: View {
     private let padding: CGFloat = 40
 
     var body: some View {
-        GeometryReader { geometry in
-            VStack(alignment: .leading, spacing: 20) {
-                Text("🔍")
-                    .font(.system(size: 60))
-                Text("buildYourNetwork")
-                    .font(.clarityBold(.title))
-                    .foregroundStyle(Color.primaryTxt)
-                Text("buildYourNetworkDescription")
-                    .foregroundStyle(Color.secondaryTxt)
-                Image.network
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
-                    .frame(width: geometry.size.width)
-                    .offset(x: -padding)
-                Spacer()
-                BigActionButton(title: "findPeople") {
-                    completion()
+        ZStack {
+            Color.appBg
+                .ignoresSafeArea()
+            ViewThatFits(in: .vertical) {
+                buildYourNetworkStack
+
+                ScrollView {
+                    buildYourNetworkStack
                 }
             }
-            .padding(padding)
-            .background(Color.appBg)
-            .navigationBarHidden(true)
         }
+        .navigationBarHidden(true)
+    }
+
+    @MainActor var buildYourNetworkStack: some View {
```

Comment:
```suggestion
     var buildYourNetworkStack: some View {
```

I'm pretty sure @MainActor here is redundant. View is marked @MainActor so all its functions and properties are too.

Reply:
I added it because the `completion()` line in the `BigActionButton` action failed to compile. I guess I could add it inside there. And sure, why not?

Reply:
Done!